x/group/keeper/grpc_query.go

```diff
@@ -356,3 +356,48 @@ func (q Keeper) Tally(ctx sdk.Context, p group.Proposal, groupId uint64) (group.
 
 	return tallyResult, nil
 }
+
+func (q Keeper) UpdateTallyOfVPEndProposals(ctx sdk.Context) error {
```

Comment:
instead of putting the whole endblocker logic in the keeper, how about putting it in a private function in `abci.go`?

IMO, the keeper only needs to expose an `IterateProposalsQueue` (similarly to what's done in gov)

Reply:
Update, but this requires `GetGroupPolicyInfo`, [`UpdateProposal`](https://github.com/cosmos/cosmos-sdk/pull/11310/files#diff-8bcf5c84d7c950183ebf3a555bd072936fee119e876e105e53f4914eee425a40R244) had to be exported from keeper.

Reply:
Ah okay, yeah. So instead of exposing 3 functions, let's do as you did before, exposing 1 function.

(would still like to see `iterateProposalsQueue` as a private method, as I will be using it)